### PR TITLE
Add JSON RPC handling for per transaction VoteBits manipulation

### PR DESCRIPTION
**Pull request by cjepson**
This adds the JSON handling for the per ticket manipulation of the
VoteBits field in wallet.

### CODE REVIEW

**Comment by alexlyp** [comment]
OK

File: dcrjson/dcrwalletextcmds.go [hunk 1]
```diff
@@ -124,6 +124,23 @@ type GetTicketsCmd struct {
...
 
+// NewGetTicketsCmd creates a new GetTicketsCmd.
+func NewGetTicketsCmd(includeImmature bool) *GetTicketsCmd {
+	return &GetTicketsCmd{includeImmature}
+}
+
+// GetTicketVoteBitsCmd is a type handling custom marshaling and
+// unmarshaling of getticketvotebits JSON wallet extension
+// commands.
+type GetTicketVoteBitsCmd struct {
```
Comment (dajohi) [inline dcrjson/dcrwalletextcmds.go]: // GetTicketVotesBitsCmd defines the getticketsvotebits JSON-RPC command.

File: dcrjson/dcrwalletextresults.go [hunk 2]
```diff
@@ -46,6 +46,13 @@ type GetTicketsResult struct {
 	Hashes []string `json:"hashes"`
 }
 
+// GetTicketVotebits models the data returned from the getticketvotebits
```
Comment (jcvernaleo) [inline dcrjson/dcrwalletextresults.go]: This should be GetTicketVoteBitsResult

File: dcrjson/dcrwalletextcmds.go [hunk 3]
```diff
@@ -124,6 +124,23 @@ type GetTicketsCmd struct {
...
+	TxHash string
+}
+
+// NewGetTicketsVoteBitsCmd creates a new GetTicketsVoteBitsCmd.
```
Comment (dajohi) [inline dcrjson/dcrwalletextcmds.go]: // NewGetTicketsVoteBitsCmd returns a new instance which can be used to issue a getticketsvotebits JSON-RPC command.

File: dcrjson/dcrwalletextcmds.go [hunk 4]
```diff
@@ -124,6 +124,23 @@ type GetTicketsCmd struct {
 	IncludeImmature bool
 }
 
+// NewGetTicketsCmd creates a new GetTicketsCmd.
```
Comment (dajohi) [inline dcrjson/dcrwalletextcmds.go]: // NewGetTicketsCmd returns a new instance which can be used to issue a gettickets JSON-RPC command.

**Comment by alexlyp** [comment]
tACK

**Pull request merged**